src/generators/css-filter.ts

```diff
@@ -63,7 +75,16 @@ export function cssFilterStyleheetTemplate(filterValue: string, reverseFilterVal
     });
 
     if (!frameURL) {
-        const [r, g, b] = applyColorMatrix([255, 255, 255], createFilterMatrix(config));
+        let r: number;
+        let g: number;
+        let b: number;
+        if (getChromeVersion() == null){
+            [r,g,b] = applyColorMatrix([255, 255, 255], createFilterMatrix(config));
+        }else if (hasChromiumIssue501582()) {
+            [r,g,b] = applyColorMatrix([0, 0, 0], createFilterMatrix(config));
```

Comment:
`createFilterMatrix(config)` could be moved to another variable (cause it is called twice in two code branches).

Reply:
Explain please?